Doc/library/argparse.rst

```diff
@@ -1853,6 +1854,11 @@ Argument groups
 
        --bar BAR  bar help
 
+   The optional, keyword-only parameters argument_default_ and conflict_handler_ 
```

Comment:
Is the comma needed here? I know that it makes a subtle semantic difference, but I do not know what is better here.

Reply:
Technically, there are two descriptors (optional and keyword-only) here so there should be a comma between them.